moto/dynamodb2/models.py

```diff
@@ -707,7 +710,9 @@ def get_item(self, table_name, keys):
         return table.get_item(hash_key, range_key)
 
     def query(self, table_name, hash_key_dict, range_comparison, range_value_dicts,
-              limit, exclusive_start_key, scan_index_forward, projection_expression, index_name=None, **filter_kwargs):
+              limit, exclusive_start_key, scan_index_forward, projection_expression, index_name=None,
+              expr_names={}, expr_values={}, filter_expression=None,
```

Comment:
Rather than having a static dictionary instance as the default for `expr_names` and `expr_values` how about just make them `None`? There's already a check for `None` for these values in `get_filter_expression`

Reply:
Ha totally agree, my ide was also complaining about the `{}`, was just toooo lazy to check `get_filter_expression`. Will fix then merge if tests pass.